### DISCUSSION TITLE
What's the use of 'embed_graph'?

### DISCUSSION THREAD

**Discussion by xkzhud** [microsoft/graphrag#1134] [Q&A]
What's the use of 'embed_graph'? The vectors it generates are not being used.You only generated the vectors, but neither the local search nor the global search uses them.  Or maybe I misunderstood. I look forward to your response.

**Comment by todap** [comment]
Did you find answer to your question? If yes please do share the answer even i have the same doubt

**Comment by xkzhud** [comment]
not yet. 

2024-10-14 13:32:01 "todap" ***@***.***> 写道：

Did you find answer to your question? If yes please do share the answer even i have the same doubt

**Comment by xkzhud** [comment]
it must be used in global search . in local search ,it's useless. iguess. 

2024-10-14 13:32:01 "todap" ***@***.***> 写道：

Did you find answer to your question? If yes please do share the answer even i have the same doubt

**Reply by todap** [reply]
I turned off the feature of generating graph embedding (this is the default parameter), i tried both, local and global query, they weren't utilised anywhere.
